corehq/apps/users/util.py

```diff
@@ -441,6 +442,18 @@ def bulk_auto_deactivate_commcare_users(user_ids, domain):
             user_doc['last_modified'] = last_modified
             user_docs_to_bulk_save.append(user_doc)
 
+    # Remove profiles
+    user_objs = []
+    for user_id in user_ids:
+        user_objs.append(CommCareUser.get_by_user_id(user_id))
+    users = prime_user_data_caches(user_objs, domain)
+    for user in users:
+        user_data = user.get_user_data(domain)
+        if user_data.profile:
+            data = user_data.to_dict()
+            user_data.update(data, '')
```

Comment:
```suggestion
            user_data.profile_id = None
```
like here:
https://github.com/dimagi/commcare-hq/blob/d1619792493f653c4936aa27d98c1f7f52023a34/corehq/apps/users/tests/test_user_data.py#L124-L126